src/compiler/checker.ts

```diff
@@ -18592,6 +18592,11 @@ namespace ts {
                     forEach(node.typeArguments, checkSourceElement);
                     if (produceDiagnostics) {
                         const symbol = getNodeLinks(node).resolvedSymbol;
+                        if (!symbol) {
+                            // There is no resolved symbol cached if the type resolved to a builtin via jsdoc type reference resolution, none of which are generic when they have no associated symbol

```

Comment:
Wrap comment, jsdoc -> JSDoc

Give a specific example of what you mean by JSDoc type reference resolution (e.g. a snippet of when this can happen)